src/redux/user/reducer.js

```diff
@@ -19,8 +20,27 @@ export const orgAdminRejected = (state) => ({
   isOrgAdmin: undefined,
 });
 
+export const writePermissionsPending = (state) => ({
+  ...state,
+  writePermissions: undefined,
+});
+
+export const writePermissionsLoaded = (state, { payload: writePermissions }) => ({
+  ...state,
+  writePermissions,
+});
+
+export const writePermissionsRejected = (state) => ({
```

Comment:
this is a duplicate of `writePermissionsPending`. You can use the same function for those two action types.